engine/classes/Elgg/Application.php

```diff
@@ -46,6 +46,7 @@ class Application {
 	 * @var string[]
 	 */
 	private static $public_services = [
+		'assets' => true,
```

Comment:
Also needs property-read for the interface, a section in guides/services.rst, and clearer guide to usage in the interface.